service-users-specs/src/test/resources/scenarios/userCreation.feature

```diff
@@ -0,0 +1,35 @@
+Feature: user creation
+
+  Background:
+    Given there is a User and Team Server
+
+  Scenario: create a user
+    Given I have a user payload
+    When I POST it to the /users endpoint
+    Then I receive a 201 status code
+
+  Scenario: cannot create twice the same user
+    Given I have a user named Johny payload
```

Comment:
can you run this scenario twice with the same output? do you use "Johny" when you create the payload, or do you generate a random prefix/suffix?

Reply:
We now add a 'random' prefix to each user. Thus we can now run them more than once.